package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@department-of-veterans-affairs/component-library",
-  "version": "3.0.3",
+  "version": "3.1.0",
```

Comment:
Bumping the minor version because the `SelectInput`, `NumberInput`, and `Date` all accept an `onBlur`. Also we have a whole new `SimpleDate`.

Arguably, this could be a major version bump because of the "breaking" change `NumberInput`—it no longer marks a field as dirty when the field is blurred. Just when it's changed.